functions/go/set-namespace-legacy/Makefile

```diff
@@ -0,0 +1,21 @@
+.PHONY: fix vet fmt test lint
```

Comment:
We should have a top-level Makefile instead of having each function have its own copy. How's the new function being built?